spock-specs/src/test/groovy/org/spockframework/junit/scheduling/SchedulingTest.groovy

```diff
@@ -0,0 +1,96 @@
+package org.spockframework.junit.scheduling
+
+import org.junit.experimental.ParallelComputer
+import org.junit.runner.Computer
+import org.junit.runner.Description
+import org.junit.runner.JUnitCore
+import org.junit.runner.notification.RunListener
+import spock.lang.Specification
+import spock.lang.Unroll
+
+class SchedulingTest extends Specification {
+    def "with sequential scheduler iterations and features should be executed sequentially"() {
+        def listener = Mock(RunListener)
+        def jUnitCore = new JUnitCore()
+        jUnitCore.addListener(listener)
+        when:
+            jUnitCore.run(Computer.serial(), SampleTest.class);
+        then:
+            1 * listener.testRunStarted(_)
+        then:
+            1 * listener.testStarted(Description.createTestDescription(SampleTest.class, "test for x=1"))
```

Comment:
`Description.createTestDescription(SampleTest.class, ...` is repeated multiple times and should be extracted into a helper method